recipes/impactutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7" %}
+
+package:
+  name: impactutils
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/earthquake-impact-utils/archive/{{ version }}.tar.gz
+  sha256: 896081c5e7528c0d5ab11415dcd52037bcc0f5af0756956fa3ef74f68a2b34e1
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python >=3.4
```

Comment:
You can just leave this a `python` and let the CI take care of the version restrictions in what it's actually building.

Reply:
impactutils does not support python 2.x, will it be clear to conda that older versions of Python are not supported?